### ISSUE TITLE
Link in the email to open the bug

### ISSUE THREAD

**Issue by Mte90** [opensupports/opensupports#235]
The email about ticket created say: `You can access to the ticket by its ticket number. `

Really? I need to do that and you cannot print the link inside the email itself? all the ticket system software add a link like GitHub, Bugzilla, Phabricator, ZenDesk.

**Comment by nickbe** [comment]
Very good idea.
On status change and receiving an email, it should be very effecient just to click the link and see the tickets

**Comment by Mte90** [comment]
I checked now and I saw that the link was added but the placeholder was broken in all the email.
Like `{{` converted to `%7B%7B` only for the link. I don't know why is happened but cleaning them fixed it.

**Comment by Mte90** [comment]
I tried again and this problem happen on the UI, maybe is a problem of sanitization.
So I have to fix by database...

**Comment by Mte90** [comment]
I am using now url like `/view-ticket/627143` but on login this show me the user login and not the admin one...
Very annoying

**Comment by nickbe** [comment]
You're right. Same problem here.  But I think Ivan already knows  about this.

**Comment by Mte90** [comment]
Ping
